### ISSUE TITLE
RapiDoc is missing `/_matrix/client/vX` prefix

### ISSUE THREAD

**Issue by aaronraimist** [matrix-org/matrix.org#1220]
For example https://matrix.org/docs/api/client-server/#get-/account/3pid is shown as  GET `/account/3pid` rather than GET `/_matrix/client/v3/account/3pid`. Clicking the try button also uses the incorrect URL https://matrix-client.matrix.org/account/3pid

**Comment by afranke** [comment]
Indeed, I noticed that last week and I intend to fix it. I thought we already had an open issue for it but apparently not. Thanks for opening it.

The reason is that each YAML file in [`matrix-doc/data/api/client-server/`](https://github.com/matrix-org/matrix-doc/tree/main/data/api/client-server) hase its own `basePath` and we drop that when we aggregate them, in favour of a single one with a value of `/`.

OpenAPI doesn’t allow per-path basePaths. We probably need to make all the individual basePaths part of the paths themselves.
